_data/sidebars/zh_lb4_sidebar.yml

```diff
@@ -0,0 +1,821 @@
+## definition of `menuitem`===========
```

Comment:
please add this sidebar to [`_config.yml`](https://github.com/mikefoxnobel/loopback.io/blob/gh-pages-zhCN/_config.yml)